README.md

```diff
@@ -1,145 +1,62 @@
 
-<p align="right">(<a href="#top">back to top</a>)</p>
+### Contributing
 
-<!-- ROADMAP -->
+Do not be intimidated or dissuaded if you have little experience with any of the tools or languages involved in this project. There are multiple ways to contribute including editing documentation, design, feature requests, logging bugs, etc. 
 
-## Roadmap
+If you are a student at North Seattle College in any field we encourage you contribute! Please see [contribution guidelines for this project](CONTRIBUTING.md) for a full outline of how to get involved.
 
-See the [open issues]("GITHUB LINK"/issues) for a full list of proposed features (and known issues).
+#### Prerequisites
 
-<p align="right">(<a href="#top">back to top</a>)</p>
+##### MacOS
+- XCode Command Line Tools `xcode-select --install`.
+- If you already have the full Xcode installed, go to menu `Xcode -> Open Developer Tool -> More Developer Tools....`.
+- Install Node.js
 
-<!-- CONTRIBUTING -->
+##### Windows
+- Install current version of Python from Microsoft Store.
+- Install Visual Studio Code.
+    - Add the Microsoft C/C++ extention.
+- Install Node.js for Windows.
+- OR Install Windows Subsystem for Linux 2 and follow the linux instructions (not recommended unless familiar with WSL already) 
 
-## Contributing
+##### Linux
 
-Contributions are what make the open source community such an amazing place to learn, inspire, and create. Any contributions you make are **greatly appreciated**.
+- Install Python > 3.7
+- Install 'make'
+- A C/C++ compiler toolchain (the GNU Compiler Collection for example)
 
-If you have a suggestion that would make this better, please fork the repo and create a pull request. You can also simply open an issue with the tag "enhancement".
-Don't forget to give the project a star! Thanks again!
-
-1. Fork the Project
-2. ```sh
-   git checkout beta
-   ```
-3.
-4. Create your Feature Branch (`git checkout -b feature/AmazingFeature`)
-5. Commit your Changes (`git commit -m 'Add some AmazingFeature'`)
-6. Push to the Branch (`git push origin feature/AmazingFeature`)
-7. Open a Pull Request
+### Installation
 
-<p align="right">(<a href="#top">back to top</a>)</p>
+STOP! -> If you plan on collaborating on this project please see the 
+[contribution guidelines](CONTRIBUTING.md). Only follow the steps below if you have direct access to this repository.
 
-<!-- LICENSE -->
+- Open a Terminal and navigate to where you want the project repository to exist
+- Clone the repo
+    ```sh
+    git clone https://github.com/NSC-Computer-Science-Club/club_website.git
+    ```
+- Navigate into the root directory
+    ```sh
+    cd club_website
+    ```
+- Install required packages
+    ```sh
+    npm install 
+    ```
+- Start server
+    ```sh
+    npm start dev
+    ```
```

Comment:
Is this for development or production? We do need info about actually running the server for development, but maybe we can wait a little bit longer until we get docs on production.

If it's for development, we should probably remove the `STOP!`. Also, `npm start dev` is meant to be `npm run dev`.